src/main/java/duke/TaskListTest.java

```diff
@@ -0,0 +1,24 @@
+package duke;
+
+import duke.task.Todo;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Represents a test driver for <code>TaskList</code>.
+ */
+public class TaskListTest {
+
+    /**
+     * Tests the addTask() method.
+     */
+    @Test
+    public void TestAddTask() {
```

Comment:
test method does not follow the `featureUnderTest_testScenario_expectedBehavior()` naming convention